### PR TITLE
minor bug fix

### PR DESCRIPTION
**Pull request by raymond-yuan**
- fixed entropy sign 
- changed entropy implementation for numeric stability

### CODE REVIEW

**Comment by raymond-yuan** [comment]
I signed it!

File: research/a3c_blogpost/a3c_cartpole.py [hunk 1]
```diff
@@ -347,15 +347,13 @@ def compute_loss(self,
...
-    entropy = tf.reduce_sum(policy * tf.log(policy + 1e-20), axis=1)
+    entropy = tf.nn.softmax_cross_entropy_with_logits_v2(labels=policy, logits=logits)
 
-    policy_loss = tf.nn.softmax_cross_entropy_with_logits_v2(labels=actions_one_hot,
-                                                             logits=logits)
+    policy_loss = tf.nn.sparse_softmax_cross_entropy_with_logits(labels=memory.actions,
+                                                                 logits=logits)
     policy_loss *= tf.stop_gradient(advantage)
-    policy_loss -= 0.01 * entropy
+    policy_loss = 0.01 * entropy
```
Comment (qlzh727) [inline research/a3c_blogpost/a3c_cartpole.py]: I think "policy_loss -= 0.01 * entropy" might be correct, overwriting the policy_loss does not make sense.

**Comment by MarkDaoust** [comment]
@qlzh727 I think that negative is there because of the `-=` later on.
I removed the double negative.

But that didn't look right. This "entropy regularization" is supposed to _discourage_ premature convergence, so it should encourage a high entropy not a low entropy.

There seems to be a lot of confusion around this point in other tutorials on the subject, because of the way the paper is written.

I think this is right. I'm merging as is. 

@raymond-yuan LMK if you disagree.

File: research/a3c_blogpost/a3c_cartpole.py (hunk 1 above)
Comment (MarkDaoust) [reply research/a3c_blogpost/a3c_cartpole.py]: You are totally correct. I fixed this before merging.

**Pull request merged**